### DISCUSSION TITLE
[Feature] Add week number to folder structure

### DISCUSSION THREAD

**Discussion by pinionless** [immich-app/immich#4260] [Feature Request]
### The feature

I store photos in folders with a date as a name. But that's too many folders for me.
I could change that to "year-month" Buts that's too many files in one folder.

I need something in between. Using week numbers would be perfect.
"UPLOAD_LOCATION/admin/{{y}}-{(week})/{{filename}}" = "UPLOAD_LOCATION/admin/2023-43/photo_1.jpg"

#EasyToImplement

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by jrasm91** [comment]
Added in #4263
